mlflow/resource_model.go

```diff
@@ -71,13 +57,20 @@ func ResourceMLFlowModel() *schema.Resource {
 			return nil
 		},
 		Read: func(ctx context.Context, data *schema.ResourceData, c *common.DatabricksClient) error {
+			var d Model
 			ad, err := api.NewModelAPI(ctx, c).Read(data.Id())
 			if err != nil {
 				return err
 			}
 
-			var d Model
-			return d.fromAPIObject(ad, s, data)
+			if err := common.StructToData(d, s, data); err != nil {
+				return err
+			}
+			data.Set("tags", ad.Tags)
+			data.Set("name", ad.Name)
```

Comment:
Wouldn't these already be set in the common.StructToData?

Reply:
values for these attributes disappear in tfstate file if removed.